tools/code_format/check_format.py

```diff
@@ -438,6 +452,10 @@ def reportError(message):
         first_word_of_prior_line = first_word
         next_word_to_check = next_word
 
+        prior_line = line
+    elif prior_line:
+      prior_line += line
```

Comment:
appending here? wouldn't this make the `not prior_line.startswith('=')` logic incorrect for the first line starting with `*`?

Reply:
oh, I can get rid of that.  Because prior_line starts out not initialized until we've hit our first line with a release note, the === lines are automatically skipped now.
I figure it's worth tracking full release note for eventual regexp magic when someone else gets annoyed at refs :-P

Reply:
Ahhh gotcha. Yep it was moreso confusing for the later version blocks, but I see there's a check for being in the current release.